client/components/Food.jsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react'
 import { Table } from 'semantic-ui-react'
 import { Link } from 'react-router-dom'
+import Label from './Label';
 
-export default function Food ({ food: { name, id } }) {
+
+export default function Food ({ food: { name, id , carbonOutput, waterUsage} }) {
   return (
     <>
       <Table singleLine>
         <Table.Header>
           <Table.Row >
             <Table.HeaderCell>
-              <Link to={`/details/${id}`}>{name}</Link>
+              <Link to={`/details/${id}`}>
+                {name}
+                <Label value={carbonOutput} type='carbon' />
+                <Label value={waterUsage} type='water' />
```

Comment:
Please use double quotes, as suggested by the [style guide](https://github.com/airbnb/javascript/tree/master/react#quotes)